File: Secao06 - Estruturas de Decisão/Exercicios em Java/Exercicio04.java
package br.com.udemy.secao06;

import java.util.Scanner;

public class Exercicio04 {

	public static void main(String[] args) {
		Scanner teclado = new Scanner(System.in);
		float altura, peso_ideal;
		char sexo;
		
		System.out.print("Informe sua altura: ");
		altura = teclado.nextFloat();
		System.out.print("Informe o sexo Homem [H] ou Mulher [M]: ");
		sexo = teclado.next().charAt(0);
		
		if (sexo == 'H' || sexo == 'h') {
			peso_ideal = (float)(72.7 * altura) - 58;
			System.out.printf("Seu peso ideal é %.2f",peso_ideal," kg.");
		}
		
		if (sexo == 'M' || sexo == 'm') {
			peso_ideal = (float)(62.1 * altura) - 58;
			System.out.printf("Seu peso ideal é %.2f",peso_ideal," kg.");
		}
		
		if (sexo != 'H' && sexo != 'h' && sexo != 'M' && sexo != 'm') {
			System.out.println("Sexo não reconhecido!");
		}
		
		teclado.close();

	}

}

File: Secao06 - Estruturas de Decisão/Exercicios em Python/Exercicio04.py
# -*- coding: utf-8 -*-
print("Calculo de peso ideal.\n")
altura = float(input("Informe sua altura: "))
sexo = input("Informe o sexo [M] Masculino ou [F] Feminino: ")

if sexo == 'M' or sexo == 'm':
    peso_ideal = (72.7 * altura) - 58
    print("Sexo Masculino com peso ideal de {0:.2f} kg.".format(peso_ideal))
elif sexo == 'F' or sexo == 'f':
    peso_ideal = (62.1 * altura) - 44.7
    print("Sexo Feminino com peso ideal de {0:.2f} kg.".format(peso_ideal))
else:
    print("Erro. Sexo incorreto!")

    
    
    
    
    